contribs/gmf/src/directives/disclaimer.js

```diff
@@ -0,0 +1,211 @@
+goog.provide('gmf.DisclaimerController');
+goog.provide('gmf.disclaimerDirective');
+
+goog.require('gmf');
+goog.require('ngeo.Disclaimer');
+goog.require('ngeo.EventHelper');
+goog.require('ngeo.LayerHelper');
+
+
+/**
+ * Provide a "disclaimer" directive for GeoMapFish that is bound to the
+ * layers added and removed from a map.
+ *
+ * Example:
+ *
+ *      <gmf-disclaimer
+ *        gmf-disclaimer-map="::ctrl.map">
+ *      </gmf-disclaimer>
+ *
+ * @htmlAttribute {ol.Map=} gmf-disclaimer-map The map.
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname gmfDisclaimer
+ */
+gmf.disclaimerDirective = function() {
+
+  return {
+    restrict: 'E',
+    scope: {
+      'map': '=gmfDisclaimerMap'
+    },
+    bindToController: true,
+    controller: 'GmfDisclaimerController',
+    controllerAs: 'dclCtrl'
+  };
+};
+
+
+gmf.module.directive('gmfDisclaimer', gmf.disclaimerDirective);
+
+
+/**
+ * @constructor
+ * @param {angular.JQLite} $element Element.
+ * @param {!angular.Scope} $scope Angular scope.
+ * @param {ngeo.Disclaimer} ngeoDisclaimer Ngeo Disclaimer service.
+ * @param {ngeo.EventHelper} ngeoEventHelper Ngeo Event Helper.
+ * @param {ngeo.LayerHelper} ngeoLayerHelper Ngeo Layer Helper.
+ * @export
+ * @ngInject
+ * @ngdoc controller
+ * @ngname GmfDisclaimerController
+ */
+gmf.DisclaimerController = function($element, $scope, ngeoDisclaimer,
+     ngeoEventHelper, ngeoLayerHelper) {
+
+  /**
+   * @type {ol.Map}
+   * @export
+   */
+  this.map;
+
+  /**
+   * @type {angular.JQLite}
+   * @private
+   */
+  this.element_ = $element;
```

Comment:
I've received counter directive...
I think that's not primordial. keep it like this.